.github/dependabot.yml

```
@@ -28,8 +28,10 @@ updates:
             - "bs58"
             - "ripemd"
         # groups are limited to 10 items
-        crypto:
+        prod:
           patterns:
+        # crypto:
+        #   patterns:
             - "bellman"
```

This may get around a limit on the number of patterns:

```suggestion
            - "bellman|red*|jubjub|group|bls12_381|blake*|secp256k1|sha2|*25519*|rand*"
```

---

Why not use `dependency-type: "production"` with `exclude-patterns` for the ECC dependencies (excluding the address dependencies).

For example:
https://github.com/ZcashFoundation/zebra/pull/7848#issuecomment-1781955833

---

Dependabot doesn't list support for setting dependency-type to production or development with cargo, and I'm not sure if it ignores patterns after the first match.